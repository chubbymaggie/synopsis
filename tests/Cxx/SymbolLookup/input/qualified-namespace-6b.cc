namespace A 
{
  namespace B 
  {
    void f1(int);
  }
}
namespace C 
{
  namespace D 
  {
    void f1(int);
  }
}
using namespace A;
using namespace C::D;
void B::f1(int){}  // okay, defines A::B::f1(int)
