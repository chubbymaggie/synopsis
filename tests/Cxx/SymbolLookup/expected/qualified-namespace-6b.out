Namespace : A
declared at line 1 in <input>
Namespace : B
declared at line 3 in <input>
Declarator : f1
declared at line 17 in <input>
Namespace : C
declared at line 8 in <input>
Namespace : D
declared at line 10 in <input>
Declarator : f1
declared at line 12 in <input>
Name : A
declared at line 1 in <input>
Name : C::D
declared at line 10 in <input>
Declarator : B::f1
declared at line 17 in <input>
